region/lowernorfair/east/The Worst Room In The Game.json

```diff
@@ -861,28 +903,53 @@
       "link": [3, 5],
       "name": "Base",
       "requires": [
-        "h_canNavigateHeatRooms",
         {"or": [
           {"and": [
-            {"or": [
-              "canTrickyJump",
-              {"enemyKill": {
-                "enemies": [["Yellow Space Pirate (standing)"]],
-                "explicitWeapons": ["ScrewAttack"]
-              }}
-            ]},
+            {"enemyKill": {
+              "enemies": [["Yellow Space Pirate (standing)"]],
+              "explicitWeapons": ["ScrewAttack"]
+            }},
             {"heatFrames": 100}
           ]},
-          {"heatFrames": 180}
+          {"heatFrames": 330}
         ]}
+      ]
+    },
+    {
+      "link": [3, 5],
+      "name": "Pirate Jump",
+      "requires": [
+        "canTrickyJump",
+        {"heatFrames": 100}
       ],
-      "note": "Spinjump over the pirate, or clear shot blocks first for more space."
+      "note": [
+        "Spinjump over the pirate.",
+        "Aiming down mid jump, , or clearing shot blocks first for more space can help but will be slower."
```

Comment:
Is this what it's supposed to say?
```suggestion
        "Spinjump over the pirate and aim down mid jump.",
        "Clearing shot blocks first for more space can help but will be slower."
```

Reply:
The downside to aim down is its more likely to land on the ledge with the pirate, so you take more heat frames.